.gitignore

```diff
@@ -1,10 +1,74 @@
-*.iml
-.gradle
-/local.properties
-/.idea
 .DS_Store
+
+.gradle
+**/build/
+!src/**/build/
```

Comment:
Why don't we want to gitignore that build folder? The `!` ignore the gitignore done by `**/build/`.

Reply:
I actually thought I removed this, my bad 🙏

Reply:
done, removed ✅